openjdk-test-common/src/test/resources/workspace/setup-profiler.bash

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+source /setup-env.d/05-utils.bash
+
+function testProfilerOn() {
+  source /setup-env.d/25-profiler-env.bash
+  if [ "$PROFILER_AGENT" != "-agentpath:/opt/cprof/profiler_java_agent.so=--logtostderr" ]; then
```

Comment:
this is a duplication of the string literal above - when we decide to change it a little bit around (e.g. increase threadcount, change logging, etc.), it will be very easy to miss this setup and can cause issues later. What do you think about refactoring the command piece into a separate file, like `profile_agent` and then you could do `"$PROFILER_AGENT" != "$(cat profile_agent)"`...?

Reply:
This file is actually used for tests only, and it's duplicated on purpose for testing.